service_test_case.py

```diff
@@ -49,11 +52,13 @@ class NioServiceTestCase(NIOTestCase):
 
     service_name = None
     auto_start = True
+    synchronous = True
 
     def __init__(self, methodName='runTests'):
         super().__init__(methodName)
         self._blocks = {}
-        self._router = ServiceTestRouter()
+        self._router = ServiceTestRouter(self.synchronous)
+        self._scheduler = Scheduler if self.synchronous else None
```

Comment:
This diff on this PR makes it looks like this isn't used anywhere. Is it used in the base `NIOTestCase`? If so, this looks like a good spot for a comment.